README.md

```diff
@@ -2,7 +2,7 @@
 
 # Prerequisites
 
-- [Node](https://nodejs.org) v6 + (v4 might work, no promises)
+- [Node](https://nodejs.org) v6, v8 is required for pa11y testing
```

Comment:
With pa11y added as a dependency, will it even install on Node 6, or is the Node 8 dependency specifically in trying to execute pa11y?

Reply:
It blows up on install on Node v6. Two options going forward:

1. Lock hard requirement to Node v8.
1. Babel transpile all tooling js

Reply:
See #136, transpiling all our dependencies sounds insane. I say we either make the addition of pa11y an add-on (Run the script/generator/etc to add pa11y, which we can do as a step of a travis run), or double-down on Node 8 being the LTS release of new projects.

Reply:
We can babel-compile just our node libraries (not the entire node_modules).

Reply:
Is this a Node 8 requirement of our node libraries or of pa11y 5? I'm confused.

Reply:
@grayside, pa11y v5 uses async/await. This feature is not available in Node <8. Our `gulp-pa11y.js` script uses pa11y v5 and thus is locked to node v8.